pkgs/applications/graphics/meshlab/wget-log

```diff
@@ -0,0 +1,11 @@
+--2019-10-10 22:26:57--  https://aur.archlinux.org/cgit/aur.git/plain/muparser.patch?h=meshlab-git
```

Comment:
seems like a logging artifact, any reason of its inclusion?

Reply:
Oops, will remove.